src/org/usfirst/frc/team4915/stronghold/subsystems/DriveTrain.java

```diff
@@ -0,0 +1,53 @@
+
+package org.usfirst.frc.team4915.stronghold.subsystems;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.RobotDrive;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import org.usfirst.frc.team4915.stronghold.Robot;
+import org.usfirst.frc.team4915.stronghold.RobotMap;
+import org.usfirst.frc.team4915.stronghold.commands.ArcadeDrive;
+
+public class DriveTrain extends Subsystem {
+
+    RobotDrive robotDrive = new RobotDrive(RobotMap.leftFrontMotor, RobotMap.leftBackMotor, RobotMap.rightFrontMotor, RobotMap.rightBackMotor);
+    public double joystickThrottle;
+
+    {
+        robotDrive.setInvertedMotor(RobotDrive.MotorType.kFrontRight, true);
+        robotDrive.setInvertedMotor(RobotDrive.MotorType.kRearRight, true);
+    }
```

Comment:
We actually did some testing and determined that the right motors were actually wired wrong, so this block isn't needed at all. My mistake